packages/core/src/lib/markdown-it/plugins/markdown-it-icons.ts

```diff
@@ -1,65 +1,97 @@
-import octicons, { IconName } from '@primer/octicons';
-import cheerio from 'cheerio';
-
+const octicons = require('@primer/octicons');
+const cheerio = require('cheerio');
 const markdownItRegExp = require('markdown-it-regexp');
 
+const ICON_REGEXP = new RegExp(
+  ':(fa[brs]|fa-brands|fa-solid|glyphicon|octicon|octiconlight|mi[forst])-'
+  + '([a-z-]+)~?([a-z-]+)?:',
+);
+
 function getOcticonIcon(iconName: string) {
-  return octicons[iconName as IconName] ?? null;
+  return octicons[iconName] ?? null;
```

Comment:
Seemed like an unrelated change to me, why is this code updated to remove `as IconName`?

Reply:
The explanation is similar to the one above regarding the mixing of CommonJS module syntax and ECMAScript 6 (ES6). However, this is indeed an unrelated refactoring of code and should be in a separate PR. The purpose of all this refactoring is to allow other modules to reuse this icon function without changing the current implementation of markdown-it-icons.ts.